src/ui/templates/search/autocomplete.hbs

```diff
@@ -1,31 +1,33 @@
-{{#if hasResults}}
-  <div class="yxt-AutoComplete">
-    {{assign 'currentSelected' sectionIndex resultIndex}}
-    {{#if promptHeader}}
-      <ul class="yxt-AutoComplete-results">
-        <li class="yxt-AutoComplete-option yxt-AutoComplete-option--promptHeader">
-          {{promptHeader}}
-        </li>
-      </ul>
-    {{/if}}
-    {{#each sections}}
-      <ul class="yxt-AutoComplete-results">
-      {{#if label}}
-        <li class="yxt-AutoComplete-resultHeader">{{label}}</li>
+<div class="yxt-AutoComplete-wrapper js-yxt-AutoComplete-wrapper" aria-expanded="{{hasResults}}">
```

Comment:
The only thing that has changed in this file is this line (and its closing) and moving the rest with tabs. I do not know why it's registering the diff so badly.